src/main/member/Interest.java

```diff
@@ -0,0 +1,26 @@
+package main.member;
+
+public enum Interest {
```

Comment:
This is not how the enums are used and created in Java. please refer to this article: https://www.w3schools.com/java/java_enums.asp and do the necessary changes.